eth/vm/forks/istanbul/transactions.py

```diff
@@ -40,3 +54,21 @@ def as_signed_transaction(self,
             r=r,
             s=s,
         )
+
+    def get_intrinsic_gas(self) -> int:
+        return _get_instanbul_intrinsic_gas(self)
+
+
+def _get_instanbul_intrinsic_gas(transaction: BaseTransaction) -> int:
```

Comment:
This fn is a duplicate of `_get_homestead_intrinsic_gas` but with a different `GAS_TXDATANONZERO` value . Should I make this a utility function with `GAS_TXDATANONZERO` as a parameter?

Reply:
Yes, better to make the original function re-usable.